package.json

```diff
@@ -7,10 +7,15 @@
     "start": "nodemon src/index.js",
     "test": "ava",
     "deploy-api": "firebase use api; firebase deploy --only functions",
-    "deploy-rules": "firebase use production; firebase deploy --only rules"
+    "deploy-rules": "firebase use staging; firebase deploy --only database",
+    "deploy-rules-production": "firebase use production; firebase deploy --only rules"
```

Comment:
shouldn't both the rules scripts use `--database`? Seems `--rules` is the old way to do it.

https://firebase.google.com/docs/cli/#deployment